sdk-cpp/CMakeLists.txt

```diff
@@ -58,19 +59,8 @@ if(${DO_PLATFORM_ID} EQUAL ${DO_PLATFORM_ID_LINUX})
     )
 
     add_library(${DO_SDK_LIB_NAME} SHARED "${sdk_source}")
```

Comment:
This is the actual definition of the target. Move it outside the if block, to line 95?

Reply:
The sdk on linux is built as a .so, but is currently a static lib on linux. Prefer leaving it here until we determine that we will ship as a dll on windows

---
In reply to: [606371871](https://github.com/microsoft/do-client/pull/19#discussion_r606371871) [](ancestors = 606371871)